src/actions/guides/getting-started/starting.cr

```diff
@@ -0,0 +1,47 @@
+class Guides::GettingStarted::StartingProject < GuideAction
```

Comment:
Could you rename this file to match the name of the action?

Reply:
yup! That's done.